README.md

```diff
@@ -152,57 +146,53 @@ The development container has been tested on the following platforms:
 
 (1) On AGX Orin Dev Kit the launch script will add ```--privileged``` and ```--group-add video``` to the docker run command for the reference applications to work. Please also make sure that the current user is member of the group video.
 
-(2) When building Holoscan SDK on AGX Orin Dev Kit from source please add the option  ```--cudaarchs all``` to the ```./run build``` command to include support for AGX Orin's iGPU.
+(2) When building Holoscan SDK on AGX Orin Dev Kit from source please add the option  ```--build-args="--cudaarchs all"``` to the ```./holohub build-container``` command to include support for AGX Orin's iGPU.
 
 # Building Operators, Applications, and Packages
 
 > _Make sure you have either launched your [development container](#container-build-recommended) or [set up your local environment](./doc/developer.md#native-build) before attempting to build Holohub components._
 
-This repository provides a convenience `run` script to abstract some of the CMake build process below.
+This repository provides a convenience `holohub` script to abstract some of the CMake build process below.
 
 Run the following to list existing components available to build:
 
   ```bash
-  ./run list
+  ./holohub run list
   ```
 
-Then run the following to build the component of your choice, using either its name or its path:
+Then run the following to build the component of your choice:
 
   ```bash
   # Build using the component name
-  ./run build <package|application|operator>
-  # Ex: ./run build endoscopy_tool_tracking
-
-  # Build using the component path
-  ./run build ./<pkg|applications|operator>/<name>
-  # Ex: ./run build ./applications/endoscopy_tool_tracking/
+  ./holohub build <package|application|operator>
+  # Ex: ./holohub build endoscopy_tool_tracking
   ```
 
-The build artifacts will be created under `./build/<component_name>` by default to isolate them from other components which might have different build environment requirements. You can override this behavior and other defaults, see `./run build --help` for more details.
+The build artifacts will be created under `./build/<component_name>` by default to isolate them from other components which might have different build environment requirements. You can override this behavior and other defaults, see `./holohub build --help` for more details.
 
 # Running Applications
 
 To list all available applications you can run the following command:
 
   ```bash
-  ./run list_apps
+  ./holohub list
```

Comment:
list_apps is not implemented.. `./holohub list` output has different sections though, maybe that's sufficient?